MatrixSDK/MXRestClient.h

```diff
@@ -155,6 +155,16 @@ typedef enum : NSUInteger
 
 #pragma mark - Registration operations
 /**
+ Check whether a username is already in use.
+ 
+ @username the user identifier to test (This value must not be nil).
```

Comment:
The terms username and user identifier (matrix user id?) are confusing.
I cannot say whether I have to pass "manu" or "@manu:matrix.org".